internal/app.go

```diff
@@ -260,6 +263,11 @@ func (a *App) cleanEvent(event *ics.VEvent) {
 			description = location + "\n" + description
 			event.SetLocation(building)
 		}
+		if navigaTUMURL := reNavigaTUM.FindString(location); navigaTUMURL != "" {
+			navigaTUMURL = strings.Trim(navigaTUMURL, "()")
+			navigaTUMURL = fmt.Sprintf("https://nav.tum.de/view/%s", navigaTUMURL)
```

Comment:
Lets use `/room` instead and name the `roomID` what it really is.

Unsure if your way of adding to the description or my way is better (likely is not relevant)

```suggestion
		if roomID := reNavigaTUM.FindString(location); roomID != "" {
			roomID = strings.Trim(navigaTUMURL, "()")
			description = fmt.Sprintf("https://nav.tum.de/room/%s\n%s", roomID, description)
```